designs/per_command_help_text.md

```diff
@@ -0,0 +1,386 @@
+                                  This option can be specified multiple times.
+
+  --s3-bucket TEXT                The name of the AWS S3 bucket where artifacts referenced in your template are uploaded.
+
+  --s3-prefix TEXT                A prefix name that is added to the artifacts name when it's uploaded 
+                                  the AWS S3 bucket. The prefix name is a path name (folder name) for the S3 bucket.
+
+  --kms-key-id TEXT               The ID of an AWS KMS key that the command uses to encrypt artifacts that are at rest
+                                  in the S3 bucket.
+
+  --role-arn TEXT                 The Amazon Resource Name (ARN) of an AWS Identity and Access Management (IAM) role
+                                  that AWS CloudFormation assumes when executing the change set.
+
+  --parameter-overrides           A string that contains AWS CloudFormation parameter overrides encoded as
+                                  key=value pairs.
+                                  
+                                  For example, 'ParameterKey=KeyPairName,ParameterValue=MyKey
+                                  ParameterKey=InstanceType,ParameterValue=t1.micro' or KeyPairName=MyKey
+                                  InstanceType=t1.micro
+
+
+  --metadata                      A map of metadata to attach to ALL the artifacts that are referenced in
+                                  your template.
+
+  --notification-arns LIST        Amazon  Simple  Notification  Service  topic.
+                                  Amazon  Resource  Names  (ARNs) that AWS CloudFormation associates with the stack.
+
+  --tags                          A list of tags to associate with the stack that is created or updated.AWS
+                                  CloudFormation also propagates these tags to resources in the stack if the resource
+                                  supports it.
+
+  --capabilities LIST             A list of capabilities that you must specify before AWS Cloudformation can create
+                                  certain stacks. Some stack templates might include resources that can affect
+                                  permissions in your AWS account, for example, by creating new AWS Identity and
+                                  Access Management (IAM) users. For those stacks, you must explicitly acknowledge
+                                  their capabilities by specifying this parameter. The only valid values are
+                                  CAPABILITY_IAM and CAPABILITY_NAMED_IAM. If you have IAM resources, you can specify
+                                  either capability. If you have IAM resources with custom names, you must specify
+                                  CAPABILITY_NAMED_IAM. If you don't specify this parameter, this action returns an
+                                  InsufficientCapabilities error.
+                                  
+  Advanced Options:
```

Comment:
I know it's common for tools to have `Advanced` options, but could the word "advanced" potentially dissuade a newer user from venturing down and trying them out. Not sure if this is a real problem or what a better term would be, mainly just thinking out loud.

Reply:
This is the right train of thought maybe it should just be named "Additional" options.